examples/solvers/douglas_rachford_primal_dual_deconvolve.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Total variation denoising using the Douglas-Rachford solver.
+
+Solves the optimization problem
+
+    min_{0 <= x <= 255}  1/2 ||x - g||_2^2 + lam || |grad(x)| ||_1
+
+where ``grad`` is the spatial gradient and ``g`` the given noisy data.
+"""
+
+import numpy as np
+import scipy
+
+import odl
+
+# Parameters
+n = 256
+filter_width = 4.0
+
+# Create a space
+space = odl.uniform_discr([0, 0], [n, n], [n, n])
+
+# Smoothing by fourier formula
+ft = odl.trafos.FourierTransform(space)
+const = filter_width**2 / 4.0**2  # by fourier transform of gaussian function
+gaussian = ft.range.element(lambda x: np.exp(-(x[0] ** 2 + x[1] ** 2) * const))
+smoothing = ft.inverse * gaussian * ft
+
+# Load cameraman data and noise
+data = space.element(np.rot90(scipy.misc.ascent()[::2, ::2], 3))
```

Comment:
conda claims that I'm using 0.17.1 (which seems to be the [newest](https://www.scipy.org/))